src/org/feedhenry/Utils.groovy

```diff
@@ -36,3 +38,9 @@ def getArtifactsDir(name) {
 def gitRepoIsDirty(untrackedFiles='no') {
     return sh(returnStdout: true, script: "git status --porcelain --untracked-files=${untrackedFiles}").trim()
 }
+
+static Closure getDate() {
```

Comment:
Thought: we could make all of the methods in here static like this, so that we wouldn't have to create an instance to use them. Are there any cons or pros to that that I'm not considering?